mne/minimum_norm/tests/test_snr.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
```

Comment:
Sure, I'll add it. I don't think many of our `test_` files have this information, BTW...